instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -57,6 +57,10 @@ def _async_call(coro: typing.Coroutine) -> asyncio.Task:
 
 
 def _response_hook(span, request: "RequestInfo", response: "ResponseInfo"):
+    # httpx < 0.20.0 url is a tuple of len 4 and not a httpx.URL
+    is_url_tuple = isinstance(request[1], tuple) and len(request[1]) == 4
+    if not is_url_tuple:
+        assert isinstance(request.url, httpx.URL)
```

Comment:
What about squashing the if into the assert? `assert is_url_tuple or isinstance(request.url, httpx.URL)`